### PR TITLE
Tech/e UI 4321 relocate find person component

### PR DESCRIPTION
**Pull request by connorpgpmcelroy**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4321

### Change description ###
Relocating all of the find-person module from role-access to common-lib

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```
Relates to - https://github.com/hmcts/rpx-xui-common-lib/pull/163

### CODE REVIEW

File: src/role-access/containers/add-exclusion/search-person/search-person.component.ts [hunk 1]
```diff
@@ -54,4 +53,8 @@ export class SearchPersonComponent implements OnInit {
   public selectedPerson(person?: Person) {
```
Comment (kuda1992) [inline src/role-access/containers/add-exclusion/search-person/search-person.component.ts:53]: Why is person optional?

File: src/role-access/containers/add-exclusion/search-person/search-person.component.ts [hunk 2]
```diff
@@ -54,4 +53,8 @@ export class SearchPersonComponent implements OnInit {
...
     this.person = person;
   }
+
+  public getDisplayName(selectedPerson: Person) {
```
Comment (kuda1992) [inline src/role-access/containers/add-exclusion/search-person/search-person.component.ts:57]: Missing return type

File: src/role-access/containers/add-exclusion/search-person/search-person.component.ts (hunk 1 above)
Comment (kuda1992) [reply src/role-access/containers/add-exclusion/search-person/search-person.component.ts:53]: And no return type

**Review by kuda1992 — CHANGES_REQUESTED** [review]
Minor changes requested

File: src/role-access/containers/add-exclusion/search-person/search-person.component.ts (hunk 1 above)
Comment (connorpgpmcelroy) [reply src/role-access/containers/add-exclusion/search-person/search-person.component.ts:53]: It was already optional. I'll see if changing it has any ill effect. Will add return type

File: src/role-access/containers/add-exclusion/search-person/search-person.component.ts (hunk 2 above)
Comment (connorpgpmcelroy) [reply src/role-access/containers/add-exclusion/search-person/search-person.component.ts:57]: Will add return type

**Review by rayliangatos — APPROVED** [review]

**Review by kuda1992 — APPROVED** [review]

**Pull request merged**